Intrepid.podspec

```diff
@@ -25,7 +25,7 @@ Pod::Spec.new do |s|
   s.source        = { :git => "https://github.com/IntrepidPursuits/swift-wisdom.git", :tag => "#{s.version}" }
   s.exclude_files = "tests/**/*"
   s.platform      = :ios
-  s.ios.deployment_target = "10.0"
+  s.ios.deployment_target = "12.0"
```

Comment:
Have not yet done a full review pass, but just initially want to discuss why the deployment target is being increased. I know firsthand of a some specific downstream apps that specifically require iOS 10 support (yes, I know that is generally overkill). So just want to think this through.

Reply:
Rolled back deployment_targets to 10.0